packages/macro/test/jsx-plural.ts

```diff
@@ -89,6 +89,42 @@ const cases: TestCase[] = [
         }} />;
       `,
   },
+  {
+    input: `
+        import { Trans, Plural} from '@lingui/macro';
+        <Plural
+              one={
+                <Trans>
+                  Care Navigator{" "}
+                  <Badge>
+                    #
+                  </Badge>
+                </Trans>
```

Comment:
Is having a 

```tsx
   <Badge>
    #
    </Badge>
```

Is important for the sake of testing this case?

Would it be enough just:

```ts
import { Trans, Plural} from '@lingui/macro';
<Plural
  one={
    <Trans>
      ...
    </Trans>}
  other={
    <Trans>
      ...
    </Trans>
  }
  value={count}
/>;
```
To reproduce the case?